hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.test.ts

```diff
@@ -0,0 +1,55 @@
+import { GameStreetActionType } from "../../models/GameSituation";
+import isBetOrRaiseExistOnFollowingActions from "./isBetOrRaiseExistOnFollowingActions";
+
+describe("isBetOrRaiseExistOnFollowingActions(gameStreetActions: GameStreetAction[], currentIndex: number)", () => {
+  it("returns true when bet is on following actions", () => {
+    expect(
+      isBetOrRaiseExistOnFollowingActions(
+        [
```

Comment:
This PlayerAuctions seems wrong for you. but it happens when PlayerAuctions[3].type change Check to Bet.